dist/assets/iframe.html

```diff
@@ -8,7 +8,7 @@
   </head>
   <body>
     <section data-main></section>
-    <script src="https://cdn.jsdelivr.net/g/lodash@4.3.0,handlebarsjs@4.0.5,jquery@2.2.0,momentjs@2.9.0,bootstrap@2.3.2"></script>
+    <script src="https://cdn.jsdelivr.net/g/lodash@2.4.2(lodash.underscore.min.js),handlebarsjs@1.3.0,jquery@2.2.4,momentjs@2.9.0,bootstrap@2.3.2"></script>
```

Comment:
Migrating a couple of functions? Probably nothing. But when your app has a million other non-compatible method calls and all you see is `'pluck' is not a function` exception in the console it makes the migration experience really frustrating :/